doc/api/modules.md

```diff
@@ -953,195 +953,31 @@ in order to be used.
 
 ## The `Module` object
 
-<!-- YAML
-added: v0.3.7
--->
-
-* {Object}
-
-Provides general utility methods when interacting with instances of
-`Module`, the `module` variable often seen in file modules. Accessed
-via `require('module')`.
-
-### `module.builtinModules`
-<!-- YAML
-added:
-  - v9.3.0
-  - v8.10.0
-  - v6.13.0
--->
-
-* {string[]}
-
-A list of the names of all modules provided by Node.js. Can be used to verify
-if a module is maintained by a third party or not.
-
-`module` in this context isn't the same object that's provided
-by the [module wrapper][]. To access it, require the `Module` module:
-
-```js
-const builtin = require('module').builtinModules;
-```
-
-### `module.createRequire(filename)`
-<!-- YAML
-added: v12.2.0
--->
-
-* `filename` {string|URL} Filename to be used to construct the require
-  function. Must be a file URL object, file URL string, or absolute path
-  string.
-* Returns: {require} Require function
-
-```js
-import { createRequire } from 'module';
-const require = createRequire(import.meta.url);
-
-// sibling-module.js is a CommonJS module.
-const siblingModule = require('./sibling-module');
-```
-
-### `module.createRequireFromPath(filename)`
-<!-- YAML
-added: v10.12.0
-deprecated: v12.2.0
--->
-
-> Stability: 0 - Deprecated: Please use [`createRequire()`][] instead.
-
-* `filename` {string} Filename to be used to construct the relative require
-  function.
-* Returns: {require} Require function
-
-```js
-const { createRequireFromPath } = require('module');
-const requireUtil = createRequireFromPath('../src/utils/');
-
-// Require `../src/utils/some-tool`
-requireUtil('./some-tool');
-```
-
-### `module.syncBuiltinESMExports()`
-<!-- YAML
-added: v12.12.0
--->
-
-The `module.syncBuiltinESMExports()` method updates all the live bindings for
-builtin ES Modules to match the properties of the CommonJS exports. It does
-not add or remove exported names from the ES Modules.
+<!-- Anchors to make sure old links find a target -->
+<a id="modules_module_builtinmodules"></a>
+<a id="modules_module_createrequire_filename"></a>
+<a id="modules_module_createrequirefrompath_filename"></a>
+<a id="modules_module_syncbuiltinesmexports"></a>
```

Comment:
It could be worthwhile including a stub link under each of these linking to the new page.